### PR TITLE
fix(server): add missing lastAccessTime field to devices response

### LINKED ISSUE #1117: GET /account/devices should return `lastConnectedAt`
The devices view displays a `last connected` field but no data is returned by the auth server to facilitate that. Presumably it should be set to the value of `lastAccessTime` from the `sessionTokens` table.

### PR DESCRIPTION
**Pull request by philbooth**
Fixes #1117.

For some reason I forgot the `lastConnectedAt` field the first time round. Here it is.

### CODE REVIEW

**Comment by philbooth** [comment]
@rfk or @dannycoates r?

File: lib/db.js [hunk 1]
```diff
@@ -434,6 +434,7 @@ module.exports = function (
             return bufferize({
               id: item.id,
               sessionToken: item.sessionTokenId,
+              lastConnectedAt: item.lastAccessTime,
```
Comment (rfk) [inline lib/db.js]: For some reason I *really* want to bikeshed the name `lastConnectedAt` here...but I'm not going to, it's pure aesthetics :-)

**Comment by rfk** [comment]
> For some reason I forgot the lastConnectedAt field the first time round

Probably because it never appeared in the spec in [1] :smile: 

[1] https://github.com/mozilla/fxa/tree/master/features/FxA-45-device-registration-api#response-3

**Comment by rfk** [comment]
LGTM, please also update the feature description over in https://github.com/mozilla/fxa for completeness.

File: lib/db.js (hunk 1 above)
Comment (philbooth) [inline lib/db.js]: Fwiw I'm happy to change it, aesthetics are important too. What would you prefer it to be?

Comment (rfk) [inline lib/db.js]: I was going to suggest `lastAccessTime` for consistency with the session data, but I really don't feel strongly about

**Comment by philbooth** [comment]
Merged with `lastAccessTime`. :grin:

**Comment by jrgm** [comment]
I have some Joi/Boom in production on that value not being integer on the response.

**Pull request merged**